src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/X509AuthorityKeyIdentifierExtension.cs

```diff
@@ -309,6 +309,10 @@ public override void CopyFrom(AsnEncodedData asnEncodedData)
         /// <exception cref="ArgumentNullException">
         ///   <paramref name="issuerName"/> is <see langword="null" />.
         /// </exception>
+        /// <exception cref="ArgumentException">
```

Comment:
I think this `<exception ...>` needs to be copied to the `byte[]` overload.

Reply:
Yeah, and the other kind of create :smile: